src/components/artifacts/ArtifactsView.tsx

```diff
@@ -177,12 +172,14 @@ class ArtifactsView extends React.Component<Props> {
       for (let info of scopedArtifactInfos) {
         if (info.isTopLevel) {
           items.push(
-            <ListItem key={info.path} button onClick={() => window.open(this.artifactURL(info.path), '_blank')}>
-              <ListItemIcon>
-                <InsertDriveFile />
-              </ListItemIcon>
-              <ListItemText primary={info.path} />
-            </ListItem>,
+            <a href={this.artifactURL(info.path)} target="_blank" rel="noopener noreferrer">
```

Comment:
I think we should use `navigate.ts` for consistency here? It will support Cmd+Click and stuff.

Reply:
I think that it would be better to use `a` tags for external links, but otherwise I'm fine with keeping the router.

Reply:
I'm looking at it with consistency in mind: we should either always use `navigate.ts` or always use `a` tag but avoid mixing them. 🤷‍♂️